backend/capellacollab/sessions/hooks/jupyter.py

```diff
@@ -65,7 +65,7 @@ def configuration_hook(
 
         volumes = self._get_project_share_volume_mounts(db, token, tool)
 
-        return environment, volumes, []
+        return environment, volumes, []  # type: ignore
```

Comment:
I suggest to be specific with what rule to ignore:

```suggestion
        return environment, volumes, []  # type: ignore[return-value]
```